jetty-servlet/src/main/java/org/eclipse/jetty/servlet/ServletHandler.java

```diff
@@ -102,36 +104,29 @@
     private boolean _ensureDefaultServlet = true;
     private IdentityService _identityService;
     private boolean _allowDuplicateMappings = false;
-
     private ServletHolder[] _servlets = new ServletHolder[0];
     private ServletMapping[] _servletMappings;
     private final Map<String, FilterHolder> _filterNameMap = new HashMap<>();
     private List<FilterMapping> _filterPathMappings;
     private MultiMap<FilterMapping> _filterNameMappings;
-
     private final Map<String, MappedServlet> _servletNameMap = new HashMap<>();
     private PathMappings<MappedServlet> _servletPathMap;
-
     private ListenerHolder[] _listeners = new ListenerHolder[0];
     private boolean _initialized = false;
-
     @SuppressWarnings("unchecked")
     protected final ConcurrentMap<String, FilterChain>[] _chainCache = new ConcurrentMap[FilterMapping.ALL];
-
     @SuppressWarnings("unchecked")
     protected final Queue<String>[] _chainLRU = new Queue[FilterMapping.ALL];
 
-    /**
-     * Constructor.
-     */
-    public ServletHandler()
+    AutoLock lock()
     {
+        return _lock.lock();
     }
 
     @Override
     public boolean isDumpable(Object o)
     {
-        return !(o instanceof Holder || o instanceof BaseHolder || o instanceof FilterMapping || o instanceof ServletMapping);
```

Comment:
This change may be correct - need to run tests to be sure - but I'd really rather you hadn't changed this in a pr which is about locking.